### PR TITLE
Update files for 5.0.0 release

### PR DESCRIPTION
**Pull request by mucles**
5.0.0 release
### Test plan
n/a

### CODE REVIEW

File: examples/multiple-executors/main.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ locals {
 
 module "networking" {
   source  = "sourcegraph/executors/aws//modules/networking"
-  version = "4.5.0" # LATEST
+  version = "5.0.0" # LATEST
 
   availability_zone = local.availability_zone
   # TODO uncomment when cutting release
```
Comment (sanderginn) [inline examples/multiple-executors/main.tf:12]: @mucles could you remove this comment and uncomment the line below? 😄

File: examples/private-single-executor/main.tf [hunk 2]
```diff
@@ -5,7 +5,7 @@ locals {
 
 module "executors" {
   source  = "sourcegraph/executors/aws"
-  version = "4.5.0" # LATEST
+  version = "5.0.0" # LATEST
 
   availability_zone                            = local.availability_zone
   executor_instance_tag                        = "codeintel-prod"
```
Comment (sanderginn) [inline examples/private-single-executor/main.tf:11]: Same in this definition

File: examples/single-executor/main.tf [hunk 3]
```diff
@@ -5,7 +5,7 @@ locals {
 
 module "executors" {
   source  = "sourcegraph/executors/aws"
-  version = "4.5.0" # LATEST
+  version = "5.0.0" # LATEST
 
   availability_zone                            = local.availability_zone
   executor_instance_tag                        = "codeintel-prod"
```
Comment (sanderginn) [inline examples/single-executor/main.tf:11]: And same here!

**Review by sanderginn — APPROVED** [review]

**Comment by Piszmog** [comment]
Double check https://github.com/sourcegraph/terraform-aws-executors/blob/master/RELEASE.md to ensure everything has been updated

**Review by Piszmog — APPROVED** [review]

**Comment by mucles** [comment]
> Double check https://github.com/sourcegraph/terraform-aws-executors/blob/master/RELEASE.md to ensure everything has been updated

Yes, looks like everything was updated to 5.0.0 👍

**Pull request merged**